public/padington.jpg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
What's this file for?

Reply:
I think this was generated when I added the image used for the basic layout. I will remove it in the next commit.